### Laelaps laelaps_control/ConfigGpio service
# 
# Configure GPIO(s).
#

Gpio[] gpio     # configure GPIO pin directions(s)
---
